README.md

```diff
@@ -3,6 +3,8 @@
 Red Hat OpenShift pipelines for certifying ISV Operator Bundles.
 
 ## Getting Started
+=======
+**To run any of the pipelines for the first time, multiple cluster resources have to be created.**
```

Comment:
You have a few merged git conflicts scattered around. Please rebase/fix your branch to remove the dupe commit.